src/components/experiment/ExperimentItem/ExperimentItem.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { ReactElement, useState } from 'react';
+import ListGroup from 'react-bootstrap/ListGroup';
+import { Modal, Button } from 'react-bootstrap';
+
+interface ExperimentItemProps {
+  url: string;
+}
+
+async function deleteExperimentFile(filename: string) {
+  await fetch(
+    `${process.env.REACT_APP_BACKEND_ADDRESS}/experiment/${filename}`,
+    { method: 'delete' }
+  );
+  window.location.reload(false);
```

Comment:
Why is this necessary?

Reply:
The deletion is handled by the ExperimentItem, and the deletion changes only the backend, the Experiment component needs to reload the list of experiments, the simplest solution seemed to be to reload the page. Alternatively, we could do something like [this](https://ourcodeworld.com/articles/read/409/how-to-update-parent-state-from-child-component-in-react) although I'd have to figure out how props work in function components, and it would ultimately have the same effect.

Reply:
That's fine, I was just curious what the purpose was. That makes sense to me as is.